bittorrent/params.go

```diff
@@ -189,7 +189,7 @@ func (qp *QueryParams) String(key string) (string, bool) {
 
 // Uint64 returns a uint parsed from a query. After being called, it is safe to
```

Comment:
Godoc doesn't fit, and apparently none of the linters caught it 🙃 My frustration with Go linters is growing

Reply:
Yeah, for some reason the golangci-lint authors believe that no one follows these conventions so they implicitly disable these lint rules.